doc/api/process.md

```diff
@@ -451,9 +451,10 @@ added: v0.1.27
 -->
 
 The `process.argv` property returns a array containing the command line
-arguments passed when the Node.js process was launched. The first element will
-be 'node', the second element will be the name of the JavaScript file. The
-remaining elements will be any additional command line arguments.
+arguments passed when the Node.js process was launched. 
+The first element will be [`process.execPath()`],
```

Comment:
No need to split all of these across lines. Just wrap them at 80 characters. Also, the sentence should end with a period, not a comma.